### DISCUSSION TITLE
Laragon php 8.3.x Imagick not found.

### DISCUSSION THREAD

**Discussion by ibrahimtrn** [leokhoa/laragon#830] [General]
"I am encountering an error like this."

`[2024-07-13 19:50:12] local.ERROR: Class "Imagick" not found {"userId":1,"exception":"[object] (Error(code: 0): Class \"Imagick\" not found at C:\\laragon\\www\\ug\\vendor\\spatie\\image\\src\\Drivers\\Imagick\\ImagickDriver.php:77)
`

I installed the Imagick extension and copied the DLL file to the ext folder. I enabled it from Laragon and the php.ini file, but it does not appear in phpinfo. What should I do?

**Comment by ppfoong** [comment]
You need to also copy all the other DLL files to the PHP folder, which is one level up from EXT folder.
